cmd/vela-cli/add.go

```diff
@@ -12,11 +12,12 @@ import (
 
 // addCmds defines the commands for creating resources.
 var addCmds = &cli.Command{
-	Name:        "add",
-	Category:    "Resource Management",
-	Aliases:     []string{"a"},
-	Description: "Use this command to add resources to Vela.",
-	Usage:       "Add resources to Vela via subcommands",
+	Name:                   "add",
+	Category:               "Resource Management",
+	Aliases:                []string{"a"},
+	Description:            "Use this command to add resources to Vela.",
+	Usage:                  "Add resources to Vela via subcommands",
+	UseShortOptionHandling: true,
```

Comment:
Here is where we enable handling of short options 👍 

Each subcommand should now have `UseShortOptionHandling` set to `true`